### ISSUE TITLE
Asset price Prediction Using QNN & Classical Hybrid Neural Network in Pennylane, Tensorflow & Amazon Braket

### ISSUE THREAD

**Issue by pratjz** [XanaduAI/QHack2022#84]
### Team Name: 

Pratjz

### Project Description: 

In this project, we will use Quantum Neural Network along with classical layers to create Hybrid Neural network using Pennylane & Tensorflow & Amazon Braket platform to present the state of the art of quantum algorithms for Financial applications, with particular focus to use cases on Continuous variable prediction problems for example Asset price prediction. We will use Boston housing data taken from the StatLib library maintained at Carnegie Mellon University, in the process we will employ various qml & ml techniques along with custom ansatz, The Ultimate goal is to Find out & demonstrate quantum advantage, be it in terms of dimensionality, accuracy, time complexity specially focusing on Scope of Quantum computing in Finance which is one of the most sought after area in Quantum computing second to Quantum Chemical simulations which itself branches out into numerous fields

Quantum Machine Learning is an emerging field that promises to solve intractable problems. There are multiple quantum machine learning algorithms that have emerged from their classical counterparts. Specially the so-called Quantum Neural Networks.

The aim of this project is to develop a clear understanding of the promises and limitations of the current state-of-the-art of quantum algorithms for machine learning, also to define directions for future research in this exciting field. And then draw the parallels to recent ‘quantum-inspired’ results, and explain the implications of these results for quantum machine learning applications. Looking for areas which might bear larger advantages for QML algorithms, like Finance & Quantum Chemical simulations

### Presentation: 

https://github.com/pratjz/Qhack2022/blob/main/Asset_Price_Prediction_Pennylane_v1.pdf

### Source code: 

https://github.com/pratjz/Qhack2022/blob/main/Asset_Price_Prediction_Pennylane_v1.ipynb

### Which challenges/prizes would you like to submit your project for?
- Amazon Braket Challenge
- Quantum Finance Challenge
- Hybrid Algorithms Challenge

**Comment by isaacdevlugt** [comment]
Hey @pratjz thank you for your submission! Just make sure that the hyperlinks direct our team directly to what is asked. It looks like your presentation hyperlink that you provided just goes to your repository, and the nested "presentation" and "notebook" links in your README link to the repository itself. There's still time to populate your repository with code, presentation material, etc. Please do so before the deadline!

Good luck!
